imports/plugins/included/marketplace/client/templates/stripeConnectSignupButton/stripeConnectSignupButton.js

```diff
@@ -47,11 +48,15 @@ Template.stripeConnectSignupButton.events({
     const businessName = shop.addressBook[0].company;
     const streetAddress = shop.addressBook[0].address1;
     const city = shop.addressBook[0].city;
-    const state = shop.addressBook[0].state;
+    const state = shop.addressBook[0].region;
     const zip = shop.addressBook[0].postal;
 
+    const user = Meteor.user() || {};
+    const { address: email } = _.first(user.emails) || {};
```

Comment:
A couple issues here:
Object destructuring doesn't work as you're intending here I don't think. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variable_names

We should be finding the default email here rather than just taking the first email available.. I'd use something like

```
const defaultEmail = user.emails.find((email) => email.provides === "default");
const defaultEmailAddress = defaultEmail.address;
```